ray-operator/controllers/ray/common/pod.go

```diff
@@ -36,10 +36,13 @@ const (
 	EnableInitContainerInjectionEnvKey = "ENABLE_INIT_CONTAINER_INJECTION"
 	NeuronCoreContainerResourceName    = "aws.amazon.com/neuroncore"
 	NeuronCoreRayResourceName          = "neuron_cores"
+	TPUContainerResourceName           = "google.com/tpu"
+	TPURayResourceName                 = "TPU"
 )
 
 var customAcceleratorToRayResourceMap = map[string]string{
 	NeuronCoreContainerResourceName: NeuronCoreRayResourceName,
+	TPUContainerResourceName:        TPURayResourceName,
```

Comment:
Can you update the unit tests?

Reply:
Added unit tests for `google.com/tpu` resources in c84ed4a.